solr/contrib/prometheus-exporter/bin/solr-exporter

```diff
@@ -86,6 +86,10 @@ for JAR in $(find "$BASEDIR"/../../dist -name 'solr-solrj-*.jar')
 do
   CLASSPATH="$CLASSPATH":"$JAR"
 done
+for JAR in $(find "$BASEDIR"/../../server/solr-webapp/webapp/WEB-INF/lib/ -name '*.jar')
+do
+  CLASSPATH="$CLASSPATH":"$JAR"
```

Comment:
Can we wildcard here?

Reply:
I can actually change most of them to wildcards. Look at the new CLASSPATH usage and let me know what you think.

Note this actually removes an error line from `find` when running in Docker (because the solrj-libs dir is deleted). 🎉

Reply:
Mike mentioned offline that the CLASSPATH `/*` notation doesn't recurse into directories, like `find` does. However we don't use that functionality. So we should be safe.